lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -462,11 +456,23 @@ def parse_json_or_nil(input)
       # Only here to please rubocop...
       return nil if input.nil?
 
-      begin
-        return JSON.parse(input)
-      rescue JSON::ParserError
-        return nil
-      end
+      input.each_codepoint do |c|
+        if c == 123
+          # left curly bracket (U+007B)
+          begin
+            return JSON.parse(input)
```

Comment:
Yeah, probably not a bad idea to do it... sometime - can you create an issue for it?